pop-cli-for-appchains/guides/spinning-up-a-polkadot-network/spinning-up-the-polkadot-network.md

```diff
@@ -4,53 +4,73 @@ description: How to spin up the Paseo Relay chain locally
 
 # Spinning up Paseo
 
-Use Pop CLI to spin up Paseo locally!
+Lets create a configuration file to launch Paseo Local:
```

Comment:
Added configuration to this file. As we would like to create a command `pop up paseo` this guide will become quickly irrelevant.